docs/local-playbook.yml

```diff
@@ -0,0 +1,28 @@
+site:
+  title: DDF Registry 
+  start_page: home::registry-intro.adoc
+  url: file:///Users/rlarsen/ddf-registry/ddf-registry/docs/build/site/
```

Comment:
❔ should this URL be here?

Reply:
no, i did not mean to commit this playbook yet